docs/content/concepts/configuration/config-schema.mdx

```diff
@@ -15,69 +15,75 @@ Run configuration allows providing parameters to jobs at the time they're execut
 
 ## Overview
 
-It's often useful to configure jobs at run time. For example, you might want someone to manually operate a deployed job and choose what dataset it operates on when they run it. In general, you should use Dagster's config system when you want the person or software that is executing a job to be able to make choices about what the job does, without needing to modify the job definition.
+It's often useful to configure Dagster jobs at runtime. For example, you might want to choose what dataset an op runs against, or provide a connection URL for a database resource. In general, you should use Dagster's config system when you want to modulate a computation at runtime without needing to modify the definition.
```

Comment:
I think it's worth mentioning software-defined assets here, because you can supply config when launching a materialization run even if you haven't defined any jobs.

Reply:
Yeah I was uncertain about this. I wanted to avoid having to list assets and ops side-by-side everytime I say anything general about config because in reality, they are using the same system. So I left it at "jobs" since asset materializations technically run through a job. But if this is hidden from the user, I see why it shouldn't be the case.

What do you think of saying "configure Dagster runs at launch"? Since runs are the public-facing common connection between jobs and asset materializations.